portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,9 +19,6 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import java.util.ArrayList;
-import java.util.List; 
-import com.google.gson.Gson;
```

Comment:
Same with this code, may just need to do another git merge query-servlet?

Reply:
same thing here, there was a merge conflict and everything showed up twice! I guess I deleted the wrong one so it comes up as a change; these imports start on line 28 now

Reply:
Ah just git being git :) All looks good then! Just make sure your imports are sorted: https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing